### PR TITLE
Gemini: Add subscription configuration

### PR DESCRIPTION
**Pull request by gbjk**
### Gemini
* Add Subscription configuration
* Upgrade test configuration

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: exchanges/gemini/gemini_websocket.go [hunk 1]
```diff
@@ -560,3 +563,28 @@ func (g *Gemini) wsProcessUpdate(result *wsL2MarketData) error {
...
+	}
+	panic(fmt.Errorf("%w: %s", subscription.ErrNotSupported, s.Channel))
+}
+
+func channelInterval(i kline.Interval) string {
+	switch i {
+	case kline.OneMin, kline.FiveMin, kline.FifteenMin, kline.ThirtyMin, kline.OneHour, kline.SixHour:
+		return i.Short()
+	case kline.OneDay:
+		return "1d"
```
Comment (shazbert) [inline exchanges/gemini/gemini_websocket.go:579]: 😆

**Review by shazbert — APPROVED** [review]
tACK.

**Review by gloriousCode — APPROVED** [review]
tACK! I don't know why I went to the effort to re-verify on Gemini with no auth sub support 😭

**Review by shazbert — CHANGES_REQUESTED** [review]
Could you please add in this change:

```diff
diff --git a/exchanges/gemini/gemini_websocket.go b/exchanges/gemini/gemini_websocket.go
index a738c2856..cecf8bba1 100644
--- a/exchanges/gemini/gemini_websocket.go
+++ b/exchanges/gemini/gemini_websocket.go
@@ -169,6 +169,7 @@ func (g *Gemini) WsAuth(ctx context.Context, dialer *websocket.Dialer) error {
        if err != nil {
                return fmt.Errorf("%v Websocket connection %v error. Error %v", g.Name, endpoint, err)
        }
+       g.Websocket.Wg.Add(1)
        go g.wsFunnelConnectionData(g.Websocket.AuthConn)
        return nil
 }
```

To resolve #1738

**Comment by gbjk** [comment]
> Could you please add in this change To resolve #1738

Fixed https://github.com/gbjk/gocryptotrader/commit/487516628

**Review by shazbert — APPROVED** [review]
Thank you very much!

**Review by thrasher- — APPROVED** [review]
tACK, thanks @shazbert for the additional fix!

**Pull request merged**